docs/resources/cce_node_v3.md

```diff
@@ -40,6 +41,16 @@ The following arguments are supported:
 
 * `cluster_id` - (Required) ID of the cluster. Changing this parameter will create a new resource.
 
+* `flavor_id` - (Required) Specifies the flavor id. Changing this parameter will create a new resource.
+
+* `availability_zone` - (Required) specify the name of the available partition (AZ). Changing this parameter will create a new resource.
+
+* `key_pair` - (Required) Key pair name when logging in to select the key pair mode. Changing this parameter will create a new resource.
+
+* `os` - (Optional) Node OS.
+  * Nodes in clusters of Kubernetes `v1.13` or later support `EulerOS 2.5`.
```

Comment:
mb would be better to do Note here?

Reply:
It's impossible to use a list in the note. Also, it's common to use a format like that, e.g `cce_cluster_v3.flavor_id`.